scalability_and_performance/ztp-deploying-disconnected.adoc

```diff
@@ -51,98 +57,171 @@ include::modules/ztp-acm-adding-images-to-mirror-registry.adoc[leveloffset=+3]
+// repo
+include::modules/ztp-upgrading-gitops-ztp.adoc[leveloffset=+1]
+
+include::modules/ztp-preparing-for-the-gitops-ztp-upgrade.adoc[leveloffset=+2]
+
+include::modules/ztp-labeling-the-existing-clusters.adoc[leveloffset=+2]
+
+include::modules/ztp-stopping-the-existing-gitops-ztp-applications.adoc[leveloffset=+2]
+
+include::modules/ztp-topology-aware-lifecycle-operator.adoc[leveloffset=+2]
+
+.Additional resources
+* For information about Topology Aware Lifecycle Operator (TALO), see xref:../scalability_and_performance/cnf-talo-for-cluster-upgrades.adoc#cnf-about-topology-aware-lifecycle-operator-config_cnf-topology-aware-lifecycle-operator[About the Topology Aware Lifecycle Operator configuration].
+
+include::modules/ztp-required-changes-to-the-git-repository.adoc[leveloffset=+2]
+
+include::modules/ztp-installing-the-new-gitops-ztp-applications.adoc[leveloffset=+2]
+
+include::modules/ztp-roll-out-the-configuration-changes.adoc[leveloffset=+2]
+.Additional resources
+* For information about creating `ClusterGroupUpgrade` CRs, see xref:../scalability_and_performance/ztp-deploying-disconnected.adoc#talo-precache-autocreated-cgu-for-ztp_ztp-deploying-disconnected[About the auto-created ClusterGroupUpgrade CR for ZTP].
+
+// Manual installation moved here
+
+include::modules/ztp-manually-install-a-single-managed-cluster.adoc[leveloffset=+1]
+
+include::modules/ztp-du-host-bios-requirements.adoc[leveloffset=+2]
+
+include::modules/ztp-configuring-a-static-ip.adoc[leveloffset=+2]
+
+include::modules/ztp-discovery-iso.adoc[leveloffset=+2]
+
+include::modules/ztp-checking-the-managed-cluster-status.adoc[leveloffset=+2]
+
+include::modules/ztp-configuring-the-cluster-for-a-disconnected-environment.adoc[leveloffset=+2]
+
+include::modules/ztp-configuring-ipv6.adoc[leveloffset=+2]
+
+include::modules/ztp-troubleshooting-the-managed-cluster.adoc[leveloffset=+2]
+
+// TALO
+
+include::modules/cnf-topology-aware-lifecycle-operator.adoc[leveloffset=+1]
+
+.Additional resources
+
+* For more information about the Topology Aware Lifecycle Operator, see xref:../scalability_and_performance/cnf-talo-for-cluster-upgrades.adoc#cnf-about-topology-aware-lifecycle-operator-config_cnf-topology-aware-lifecycle-operator[About the Topology Aware Lifecycle Operator].
+
+include::modules/cnf-topology-aware-lifecycle-operator-autocreate-cgu-cr-ztp.adoc[leveloffset=+1]
+
+include::modules/cnf-topology-aware-lifecycle-operator-preparing-for-updates.adoc[leveloffset=+2]
+
+include::modules/cnf-topology-aware-lifecycle-operator-platform-update.adoc[leveloffset=+2]
+
+include::modules/cnf-topology-aware-lifecycle-operator-operator-update.adoc[leveloffset=+2]
 
-include::modules/ztp-validating-the-generation-of-policy-crs.adoc[leveloffset=+2]
+include::modules/cnf-topology-aware-lifecycle-operator-operator-and-platform-update.adoc[leveloffset=+2]
```

Comment:
These should not be under the module cnf-topology-aware-lifecycle-operator-autocreate-cgu-cr-ztp. There are end-to-end procedures for upgrading the restricted network cluster and operators with TALO which should be under a new title

Reply:
@amolnar-rh thanks Alexandara. Now "End-to-end procedures for updating clusters in a disconnected environment" is under "Updating managed policies with the Topology Aware lifecycle opeator" which is still not ideal. They should be the same level.

Reply:
@yliu127 @Missxiaoguo Updated.